kafka/README.md

```diff
@@ -78,11 +77,27 @@ After you've configured `kafka.yaml`, [restart the Agent][10] to begin sending K
           #    pattern: \d{4}\-(0?[1-9]|1[012])\-(0?[1-9]|[12][0-9]|3[01])
     ```
 
-5. [Restart the Agent][10].
+5. [Restart the Agent][8].
+
+#### Containerized
+
+##### Metric collection
+
+For containerized environments, see the [Autodiscovery with JMX][10] guide.
+
+##### Log collection
+
+**Available for Agent v6.5+**
+
+Collecting logs is disabled by default in the Datadog Agent. To enable it, see [Docker log collection][11].
```

Comment:
A small but important comment here -- since we support different types of containerized environments, such as Docker, Kubernetes and ECS, we need to refer the user to a page where they can get the right directions for their environment. Here's the Kubernetes log collection for example: https://docs.datadoghq.com/agent/kubernetes/daemonset_setup/?tab=k8sfile#log-collection

We've actually seen multiple customer tickets this year where customers used the wrong instructions for their containerized environment like here

Reply:
Let's create a card an update this in another PR once all AD instructions update have been merged.